Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners.yml

```
@@ -0,0 +1,21 @@
+args:
+- description: List of potential service owners
+  isArray: true
+  name: owners
+comment: Recommend most likely service owners from those surfaced by Cortex ASM Enrichment.
+commonfields:
+  id: RankServiceOwners
+  version: -1
+dockerimage: demisto/python3:3.10.11.54132
+enabled: true
+name: RankServiceOwners
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags: []
+type: python
+fromversion: 6.2.0
```

```suggestion
fromversion: 6.5.0
```

Can you change this to 6.5.0 without error?